tests/test_import.py

```diff
@@ -1,11 +1,12 @@
 import subprocess
+import sys
 
 
 def test_import_smolagents_without_extras():
-    # Run the import statement in an isolated virtual environment
-    result = subprocess.run(
-        ["uv", "run", "--isolated", "--no-editable", "-"], input="import smolagents", text=True, capture_output=True
-    )
+    # Create a new Python process to test the import
+    cmd = [sys.executable, "-c", "import smolagents"]
+    result = subprocess.run(cmd, capture_output=True, text=True)
+
```

Comment:
I think this change breaks the purpose of the test:
- test import of smolagents where the lib has been installed without extras.

The new code tests the import of smolagents in current Python environment, where the lib was installed with all the extras.